coveralls.json

```diff
@@ -20,6 +20,9 @@
   },
 
   "skip_files": [
-    "test"
+    "test",
+    "lib/phoaw/application.ex",
+    "lib/phoaw_web.ex",
+    "lib/phoaw_web/views/error_helpers.ex"
```

Comment:
why skip this files?

Reply:
We don't use this file yet, so we didn't have test for this file